### PR TITLE
Bigdl example: Semantic Segmentation

### PR DESCRIPTION
**Pull request by leonardozcm**

### CODE REVIEW

**Comment by yangw1234** [comment]
let's change the pytorch examples all in `example/pytorch`

**Comment by leonardozcm** [comment]
> let's change the pytorch examples all in `example/pytorch`

OK, I will apply these changes here.

File: python/nano/example/pytorch/semantic_segmentation/semantic_segmentation.py [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+    # 3 START TRAINING
+    # ------------------------
+    trainer.fit(model)
+
+
+if __name__ == "__main__":
+    cli_lightning_logo()
+    parser = ArgumentParser(add_help=False)
+    parser = SegModel.add_model_specific_args(parser)
+    parser.add_argument("--ipex", action="store_true",
```
Comment (yangw1234) [inline python/nano/example/pytorch/semantic_segmentation/semantic_segmentation.py:317]: The original example should not have this right?

Comment (leonardozcm) [reply python/nano/example/pytorch/semantic_segmentation/semantic_segmentation.py:317]: Yes, I will remove it.

**Comment by leonardozcm** [comment]
http://10.112.231.51:18888/view/ZOO-PR/job/ZOO-PR-NanoTests/14/

**Comment by yangw1234** [comment]
LGTM

**Pull request merged**